temporal_sqlalchemy/bases.py

```diff
@@ -194,22 +240,13 @@ def date_modified(self):
 
     @contextlib.contextmanager
     def clock_tick(self, activity: TemporalActivityMixin = None):
-        warnings.warn("clock_tick is going away in 0.5.0",
-                      PendingDeprecationWarning)
-        """Increments vclock by 1 with changes scoped to the session"""
-        if self.temporal_options.activity_cls is not None and activity is None:
-            raise ValueError("activity is missing on edit") from None
-
-        session = orm.object_session(self)
-        with session.no_autoflush:
-            yield self
-
-        if session.is_modified(self):
-            self.vclock += 1
+        warnings.warn("clock_tick is deprecated, assign an activity directly",
+                      DeprecationWarning)
+        if self.temporal_options.activity_cls:
+            if not activity:
+                raise ValueError
```

Comment:
Can we put an error message here please? It'll make debugging easier. I think the original error message might suffice.

Reply:
👍 should probably be:

```python

raise ValueError("activity is missing on edit") from None
```